docs/molgenis/use_usingpyclient.md

```diff
@@ -0,0 +1,281 @@
+Raises the `TokenSigninException` when the client is already signed in with a username/password combination.
+
+| argument | type | description | required | default |
+|----------|------|-------------|----------|---------|
+| token    | str  | the token   | True     |         |
+
+### get_schema_metadata
+```python
+client.get_schema_metadata(name='My Schema')
+```
+Retrieves a schema's metadata, including the names of the tables, the metadata of the columns in the tables, schema settings, etc.
+If no `name` is supplied the metadata for the default schema is queried, if applicable. 
+Throws the `NoSuchSchemaException` if the user does not have at least _viewer_ permissions or if the schema does not exist.
+
+| argument | type | description          | required | default |
+|----------|------|----------------------|----------|---------|
+| name     | str  | the name of a schema | False    | None    |
+
+### get
+```python
+client.get(table='Data Table', schema='My Schema', as_df=True)
+```
+Retrieves data from a table on a schema and returns the result either as a list of dictionaries or as a pandas DataFrame (as pandas is used to parse the response).
+Throws the `NoSuchSchemaException` if the user does not have at least _viewer_ permissions or if the schema does not exist.
+
+| argument | type | description                                                                    | required | default |
+|----------|------|--------------------------------------------------------------------------------|----------|---------|
+| table    | str  | the name of a table                                                            | True     | None    |
+| schema   | str  | the name of a schema                                                           | False    | None    |
+| as_df    | bool | if true: returns data as pandas DataFrame <br/> else as a list of dictionaries | False    | False   |
+
+### export
+```python
+client.export(schema='My Schema', table='Data Table', fmt='csv')
+```
+Exports data from a schema to a file in the desired format.
+If the table is specified, only data from that table is exported, otherwise the export contains all tables on the schema.
+If all tables from a schema are exported with given format `csv`, the data is exported as a zip file containing a csv file of each table.
+Throws the `NoSuchSchemaException` if the user does not have at least _viewer_ permissions or if the schema does not exist.
+
+| argument | type | description                               | required | default |
+|----------|------|-------------------------------------------|----------|---------|
+| table    | str  | the name of a table                       | False    | None    |
+| schema   | str  | the name of a schema                      | False    | None    |
+| fmt      | str  | the output format, either `csv` or `xlsx` | False    | `csv`   |
+
+
+### save_schema
+```python
+client.save_schema(table='Data Table', name='My Schema',  
```

Comment:
For a later point:
What happens if users uses a file AND data, the function does not check for that right?

Reply:
No, indeed. If both are given only the file is considered